src/main/java/com/netflix/imflibrary/st0377/header/MCALabelSubDescriptor.java

```diff
@@ -31,12 +31,12 @@
      */
     public static abstract class MCALabelSubDescriptorBO extends SubDescriptorBO{
 
-        @MXFProperty(size=16) protected final UL mca_label_dictionary_id = null;
-        @MXFProperty(size=16) protected final byte[] mca_link_id = null; //UUID type
+        @MXFProperty(size=16) public final UL mca_label_dictionary_id = null;
```

Comment:
None of these fields need to be _public_ do they? You have created public accessors for them further down.

Reply:
Good catch. Fixed.